src/lib/axios.js

```diff
@@ -24,5 +26,9 @@ module.exports = function (req, res, next) {
     req.axios.defaults.headers.common["x-scenario-id"] = req.scenarioIDHeader;
   }
 
+  if (req?.headers["forwarded"] && req.axios?.defaults?.headers?.common) {
+    req.axios.defaults.headers.common["x-forwarded-for"] = userIpAddress(req);
```

Comment:
One other bug with this current implementation is that if the `forwared` header is present, but in a different format, the `x-forwarded-for` header is passed through as null, e.g. `"x-forwarded-for":null`

This header should only be added if it is not null, either by assigning via a variable, or a conditional include like https://github.com/alphagov/di-ipv-cri-address-front/blob/main/src/app.js#L50